app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.kt

```diff
@@ -149,9 +164,24 @@ class SearchActivity : BindingBaseActivity<SearchActivityComponent>() {
 
     companion object {
 
-        @JvmStatic fun open(context: Context) {
+        @JvmStatic
+        fun open(context: Context) {
             Intent(context, SearchActivity::class.java)
                     .apply { context.startActivity(this) }
         }
     }
+
+    private inner class SearchPagerAdapter(fm: FragmentManager?) : FragmentPagerAdapter(fm) {
+        override fun getItem(position: Int): Fragment = if (position == 0) SearchFragment.create() else MapFragment.create()
+
+        override fun getCount(): Int = 2
+
+        override fun getPageTitle(position: Int): CharSequence =
+                when (position) {
+                    0 -> getString(R.string.search_tab_results)
+                    1 -> getString(R.string.search_tab_map)
+                    else -> ""
```

Comment:
This is a error condition, correct? If so then throw an `IllegalArgumentException` with an explanation from here.